plugins/tiddlywiki/multiwikiserver/templates/add-user-modal.tid

```diff
@@ -0,0 +1,40 @@
+title: $:/plugins/tiddlywiki/multiwikiserver/templates/add-user-modal
+
+\define add-user-actions()
+<$action-sendmessage $message="tm-server-request"
```

Comment:
I may be missing something obvious, but where is the definition of the tm-server-request message handler?